ansible/roles/rems-watcher/templates/restart_rems.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+FILE=/var/log/messages
+LINEFILE=/tmp/rems_error_track/line-track.txt
+myPath=/tmp/rems_error_track
+
+# This script restarts rems if it is not running properly
```

Comment:
Move this comment describing script to right after opening line `#!/bin/bash`, separated by empty lines.